### ISSUE TITLE
The "@serverless-stack/aws-lambda-ric" dependency is not resolved reliably

### ISSUE THREAD

**Issue by octogonz** [serverless-stack/serverless-stack#1584]
While testing the tutorial project in https://github.com/microsoft/rushstack/pull/3282,  we encountered an issue where `@serverless-stack/aws-lambda-ric` is not being resolved correctly. The error looks like this:

```
Debug session started. Listening for requests...
a65a98b3-5b98-41d6-80b2-2e8f50daaa26 REQUEST repro-serverless-stack-tu-ApiLambdaGETB1714EF3-MjSo5wKL9F0a [src/lambda.handler] invoked by API GET /
2022-03-29T10:20:13.693Z    undefined   INFO    Executing '/Users/nobody/repro/rushstack/build-tests-samples/heft-serverless-stack-tutorial/.sst/artifacts/repro-serverless-stack-tutorial-my-stack-Api-Lambda_GET_-/src/lambda.handler' in function directory '/Users/nobody/repro/rushstack/build-tests-samples/heft-serverless-stack-tutorial'
- /Users/nobody/.npm/_npx/51045/lib/node_modules/aws-lambda-ric/bin/index.jslient/index.jslient.jse-client.node'
```

@frekje-wm was able to fix the issue by adding `"@serverless-stack/aws-lambda-ric": "^2.0.12"` as a direct dependency of the project. The solution may be similar to PR https://github.com/serverless-stack/serverless-stack/pull/1582

**Comment by octogonz** [comment]
I was able to repro this problem on Windows. The error looks like this:
```
Debug session started. Listening for requests...
90d608d2-47aa-4408-9964-c8916bec7502 REQUEST prod-heft-serverless-stack-tu-ApiLambdaGETB1714EF3-Q2WJdPoQrWor [src/lambda.handler] invoked by API GET /
'.' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! 
code ELIFECYCLE
npm ERR! errno 1
npm
ERR! aws-lambda-ric@2.0.0 preinstall: `./scripts/preinstall.sh`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the aws-lambda-ric@2.0.0 preinstall script.
```
The workaround of adding `@serverless-stack/aws-lambda-ric` as a direct dependency also works on Windows.

**Comment by fkjellberg** [comment]
PR: https://github.com/serverless-stack/serverless-stack/pull/1588

**Comment by fwang** [comment]
Fixed in v0.69.6 and v1.0.0-beta.20

@octogonz @fkjellberg Sorry about the delay in merging the fix. We were in the middle of restructuring the repo for v1 launch. :crossed_fingers:
